src/alembic_utils/pg_policy.py

```diff
@@ -0,0 +1,153 @@
+# pylint: disable=unused-argument,invalid-name,line-too-long
+from __future__ import annotations
+
+from typing import List
+
+from parse import parse
+from sqlalchemy import text as sql_text
+
+from alembic_utils.exceptions import SQLParseFailure
+from alembic_utils.replaceable_entity import ReplaceableEntity, normalize_whitespace
+
+
+class PGPolicy(ReplaceableEntity):
+    """A PostgreSQL Function compatible with `alembic revision --autogenerate`
+
+    **Parameters:**
+
+    * **schema** - *str*: A SQL schema name
+    * **signature** - *str*: A SQL policy name and tablename, separated by "."
+    * **definition** - *str*:  The definition of the policy, incl. permissive, for, to, using, with check
+
+    """
+
+    is_replaceable = False
```

Comment:
Rather than creating an `is_replaceable` flag, please populate the `to_sql_statement_create_or_replace` with a statement that first drops the policy, and then re-creates it